CMakeLists.txt

```diff
@@ -133,7 +135,7 @@ set(CMAKE_CXX_FLAGS         "${EVT_COMMON_FLAGS} \
 # TODO: Check for platform to decide which linker script to use
 set(CMAKE_EXE_LINKER_FLAGS  "-mfloat-abi=hard -specs=nano.specs -specs=nosys.specs \
                             -T ${PROJECT_SOURCE_DIR}/libs/HALf3/STM32F302C8Tx_FLASH.ld \
-                            -lc -lm -lnosys")
+                            -lc -lm -lnosys -u _printf_float")
```

Comment:
Be careful that we don't abuse this, it's disabled by default for a reason. For ADC readings, multiplying by 1000 and static casting to an integer keeps your precision without the overhead for pretty printouts, both from a code size and execution time perspective.

Reply:
Oh great point. In the future u can move this to the compile flags of the specific sample